Jenkinsfile

```diff
@@ -22,55 +22,61 @@ pipeline {
                 configFileProvider([configFile(fileId: 'maven-settings-with-deploy-snapshot', variable: 'MAVEN_SETTINGS')]) {
                     script {
                         def mvnHome = tool 'Maven'
-                        sh "${mvnHome}/bin/mvn clean install -s $MAVEN_SETTINGS -Dmaven.test.failure.ignore=true || true"
+                        sh "${mvnHome}/bin/mvn clean install -pl :infinispan-core -am -PtraceTests -s $MAVEN_SETTINGS -Dmaven.test.failure.ignore=true || true"
                         junit testDataPublishers: [[$class: 'ClaimTestDataPublisher']], testResults: '**/target/*-reports/*.xml'
```

Comment:
The archive step should be here IMO. BTW, why are you using `'**/infinispan*log.gz'`? Do you expect them to be gzipped?